docs/help-topics/stonesoup-learnmore/stonesoup-add-a-component/metadata.yml

```diff
@@ -0,0 +1,2 @@
+kind: HelpTopic
```

Comment:
Based on the help-topic [readme](https://github.com/RedHatInsights/quickstarts/blob/main/docs/help-topics/README.md) you should create a new directory for every help topic. So there shouldn't be a `stonesoup-learnmore` directory. You should have `stonesoup-add-a-component` directly as the directory.

Reply:
I added a stonesoup-learnmore dir just like Jake added a stonesoup-whatsnext dir. This is per Danny Philips' instructions.

Reply:
I tried using whats next help topics added by Jake in the UI. It doesn't work and I am guessing this is the main issue.